pkg/agent/datamodel/mocks.go

```diff
@@ -3,77 +3,6 @@
 
 package datamodel
 
-import (
-	"github.com/Azure/go-autorest/autorest/to"
-	"github.com/google/uuid"
-)
-
-// CreateMockContainerService returns a mock container service for testing purposes.
-func CreateMockContainerService(containerServiceName, orchestratorVersion string, masterCount, agentCount int, certs bool) *ContainerService {
```

Comment:
we're sure we don't have any further usage of this?

Reply:
Yep there are no other calls to it in the repo